CallsAndSMS/app/src/main/AndroidManifest.xml

```diff
@@ -1,57 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="nisrulz.github.example.callsandsms">
-
-    <!--Required to make the call directly-->
-    <uses-permission android:name="android.permission.CALL_PHONE" />
-
-    <!--Required to monitor call states-->
-    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
-
-    <!--To listen to outgoing calls-->
-    <uses-permission android:name="android.permission.PROCESS_OUTGOING_CALLS" />
-
-    <!--To send sms directly -->
-    <uses-permission android:name="android.permission.SEND_SMS" />
+  package="nisrulz.github.example.callsandsms">
```

Comment:
Refactor for package name `github.nisrulz.example.*`

Reply:
Fixed